### DISCUSSION TITLE
在青龙上面，环境检测出现下面这个要怎么处理？

### DISCUSSION THREAD

**Discussion by moshi888** [0-8-4/miui-auto-tasks#213] [问与答｜Q&A]
/ql/shell/otask.sh: line 140: cd: 0-8-4_miui-auto-tasks_master: No such file or directory
python3: can't open file '/ql/data/scripts/miuitask.py': [Errno 2] No such file or directory

**Comment by Night-stars-1** [comment]
给出你青龙里这个任务的相关配置
